### ISSUE TITLE
Support numeric entities

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#19]
```
What steps will reproduce the problem?
1. Use RSS with numeric entities other than quotes, and umlaut chars

What is the expected output? Entities converted
What do you see instead?  Characters not there


This is fixed on version 1.9

```

Original issue reported on code.google.com by `ibunto...@gmail.com` on 14 Aug 2007 at 8:59

**Comment by GoogleCodeExporter** [comment]
```
This is fixed on version 1.9.
```

Original comment by `ibunto...@gmail.com` on 6 Sep 2007 at 1:57
- Added labels: **Milestone-Release1.9**, **Type-Patch**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This was verified with the release
```

Original comment by `ibunto...@gmail.com` on 15 Aug 2008 at 9:12
- Changed state: **Verified**
